lib/truncate_html/html_truncator.rb

```diff
@@ -7,11 +7,20 @@ def initialize(original_html, options = {})
       @omission        = options[:omission]     || TruncateHtml.configuration.omission
       @word_boundary   = (options.has_key?(:word_boundary) ? options[:word_boundary] : TruncateHtml.configuration.word_boundary)
       @break_token     = options[:break_token] || TruncateHtml.configuration.break_token || nil
+      @break_tokens    = options[:break_tokens] || TruncateHtml.configuration.break_tokens || []
+      @break_tokens    << @break_token
```

Comment:
since `@break_token` can be set to nil on line 9, you may want to add a protection here so you're not shoveling a nil value into your `@break_tokens` array. maybe something like `@break_tokens << @break_token if @break_token`.  

if there are any nil values in `@break_tokens`, the code will throw an error at [line 83](https://github.com/Bugagazavr/truncate_html/blob/master/lib/truncate_html/html_truncator.rb#L83):
``` NoMethodError - undefined method '<<' for nil:NilClass```